docs/working-with-projects/project-configuration/create-robot-accounts.md

```diff
@@ -13,33 +13,33 @@ You can create robot accounts to run automated operations. Robot accounts have t
 1. Log in to the Harbor interface with an account that has at least project administrator privileges.
 1. Go to **Projects**, select a project, and select **Robot Accounts**.
 
-    ![Robot accounts](../../../img/add-robot-account.png)
+    ![Robot accounts](../../img/add-robot-account.png)
 
 1. Click **New Robot Account**.
 1. Enter a name and an optional description for this robot account.
-1. Set expiration time for this robot account. If not set, the expiration time of system configuration will be used for this robot account.
+1. Set expiration time for this robot account, select checkbox **Never Expired** if you want to create a permanent robot account. If not set, the expiration time of system configuration will be used for this robot account. 
 1. Grant permission to the robot account to push images and to push and pull Helm charts.
 
     Robot accounts can always pull images, so you cannot deselect this option.
    
-    ![Add a robot account](../../../img/add-robot-account-2.png)
+    ![Add a robot account](../../img/add-robot-account-2.png)
 
 1. Click **Save**.
 1. In the confirmation window, click **Export to File** to download the access token as a JSON file, or click the clipboard icon to copy its contents to the clipboard.
 
-   ![copy_robot_account_token](../../../img/copy-robot-account-token.png)
+   ![copy_robot_account_token](../../img/copy-robot-account-token.png)
 
    {{< important >}}
    Harbor does not store robot account tokens, so you must either download the token JSON or copy and paste its contents into a text file. There is no way to get the token from Harbor after you have created the robot account.
    {{< /important >}}
 
    The new robot account appears as `robot$account_name` in the list of robot accounts. The `robot$` prefix makes it easily distinguishable from a normal Harbor user account.
 
-   ![New robot account](../../../img/new-robot-account.png)
+   ![New robot account](../../img/new-robot-account.png)
```

Comment:
‘../../../' might be correct, it's always one level more than correct directory structure, please make sure